breacharbiter.go

```diff
@@ -152,10 +152,12 @@ func (b *breachArbiter) start() error {
 	brarLog.Tracef("Starting breach arbiter")
 
 	// Load all retributions currently persisted in the retribution store.
-	breachRetInfos := make(map[wire.OutPoint]retributionInfo)
+	var breachRetInfos map[wire.OutPoint]retributionInfo
 	if err := b.cfg.Store.ForAll(func(ret *retributionInfo) error {
 		breachRetInfos[ret.chanPoint] = *ret
 		return nil
+	}, func() {
+		breachRetInfos = make(map[wire.OutPoint]retributionInfo)
```

Comment:
nit (maybe a large change to make at this point): since this method is always run first, would it make sense to have it be the first argument?

Reply:
I wanted to make the changelist minimal, and imho it's more readable to have the tx logic in the first place.